maxim/tests/test_logger_langchain_03x.py

```
@@ -871,6 +877,353 @@ def capitalize_text(text: str) -> str:
+            }
+        )
+
+        # Get the path to the test image file
+        test_files_dir = Path(__file__).parent / "files"
+        image_path = test_files_dir / "png_image.png"
+
+        # Read the file as raw bytes
+        file_bytes = image_path.read_bytes()
+
+        # Use a public image URL as second image
+        image_url = "https://upload.wikimedia.org/wikipedia/commons/thumb/4/47/PNG_transparency_demonstration_1.png/300px-PNG_transparency_demonstration_1.png"
```

Ruff (0.14.10)

Local variable `image_url` is assigned to but never used

Remove assignment to unused variable `image_url`

(F841)